test/jest/Backup.spec.ts

```diff
@@ -0,0 +1,173 @@
+import fs from 'fs';
+import path from 'path';
+import Backup from '../../lib/backup/Backup';
+
+const removeDir = (dir: string) => {
+  if (fs.existsSync(dir)) {
+    fs.readdirSync(dir).forEach((file) => {
+      fs.unlinkSync(`${dir}/${file}`);
+    });
+  }
+
+  fs.rmdirSync(dir);
+};
+
+const wait = (ms: number) => {
```

Comment:
There's a utils `setTimeoutPromise` for this you can import and use in the same way instead of this function.